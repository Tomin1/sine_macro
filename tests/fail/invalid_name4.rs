/*
 * Copyright (c) 2025 Tomi Lepp√§nen
 * SPDX-License-Identifier: MIT
 */

use sine_macro::sine_wave;

fn main() {
    let _wave = sine_wave!(rate: 1000, frequency: 100, invalid: 10);
}
