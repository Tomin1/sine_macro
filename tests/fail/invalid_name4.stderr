error: unexpected identifier
 --> tests/fail/invalid_name4.rs:9:56
  |
9 |     let _wave = sine_wave!(rate: 1000, frequency: 100, invalid: 10);
  |                                                        ^^^^^^^
